src/our-works.html

```diff
@@ -4,155 +4,174 @@
+<aside class="menu is-hidden-desktop navbar-menu-mobile is-opacity">
+  <ul class="menu-list">
+    <li class="menu-list-item"><a href="./security.html">What We Do</a></li>
+    <li class="menu-list-item"><a href="./our-works.html">Our Works</a></li>
+    <li class="menu-list-item"><a>The Platform</a></li>
+    <li class="menu-list-item">
+      <button class="button btn-o btn-o-white is-uppercase">Contact us</button>
+    </li>
+  </ul>
+</aside>
+<div class="top-section">
+  <nav class="navbar navbar-fixed" role="navigation" aria-label="main navigation">
+    <div class="container">
+      <div class="navbar-brand">
+        <a class="navbar-item navbar-brand-logo" href="/">
+          <img src="./images/logo/100x86.png">
+        </a>
+        <a role="button" class="navbar-burger" aria-label="menu" aria-expanded="false">
+          <span aria-hidden="true"></span>
+          <span aria-hidden="true"></span>
+          <span aria-hidden="true"></span>
+        </a>
+      </div>
+      <div class="navbar-menu">
+        <div class="navbar-end">
+          <a href="./security.html" class="navbar-item">What We Do</a>
+          <a href="./our-works.html" class="navbar-item">Our Works</a>
+          <a class="navbar-item">The Platform</a>
+          <button class="navbar-item button btn-o btn-o-white is-uppercase">Contact us</button>
         </div>
       </div>
     </div>
-    <div class="content white">
-      <div class="chapter-line">
-        <div class="left-line-black"></div>
-        <div>WHAT WE DO</div>
+  </nav>
+  <div class="container">
+    <div class="top-section-content top-section-content-large">
+      <h2 class="title title-lg is-spaced top-section-title-lg">Active supporters of a better society. <br class="is-hidden-desktop"> Today.</h2>
+      <div class="subtitle subtitle-left-border top-section-subtitle">The cyberpunk movement was developed by a group of people who hardly believed in a World<br class="is-hidden-mobile"> inwhich privacy and cryptography were foundamental components for reinvent the modern<br class="is-hidden-mobile"> society.</div>
+      <div class="scroll-down">
+        <a href="#info-section1" class="scroll-down-link">
+          <span class="subtitle subtitle-link scroll-down-text">SCROLL DOWN</span>
+          <button class="button icon scroll-down-button">
+            <i class="scroll-down-button-icon"></i>
+          </button>
+        </a>
       </div>
-
-      <div class="intro-panel">
-        <div class="intro-panel-content">
-          <h1>Why if we think help million people instead a table of six?</h1>
-          <div class="intro-text">
-            Our passion for crypto philosophy led us to spread our vision through each work we do. Doesn't matter if the goal is design a new product or write a new guide, we really believe in the crypto business revolution.
-          </div>
-        </div>
+    </div>
+  </div>
+</div>
+<div id="info-section1" class="info-section info-section-light">
```

Comment:
I gave this id, because I thought that there might be similar blocks on the page. Sorry if I gave  a name that was not very suitable. Do you have a more suitable variant?